src/Controls/tests/Core.UnitTests/TypedBindingFromExpressionUnitTests.cs

```diff
@@ -0,0 +1,1704 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Maui.Controls.Internals;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.Maui.Controls.Core.UnitTests
+{
+	using StackLayout = Microsoft.Maui.Controls.Compatibility.StackLayout;
+
+
+	public class TypedBindingFromExpressionUnitTests : BindingBaseUnitTests
```

Comment:
Were the unit tests in here copied from another file? I wonder if the tests could share test cases, and abstract out the binding creation part.

Reply:
Yes, I copied the tests from `TypedBindingUnitTests`. I'll look into it.

Reply:
I updated the original tests to always run twice - once with explicitly constructed `TypedBinding` and once with the binding created from expression.